feature-libs/customer-ticketing/occ/adapters/occ-customer-ticketing.adapter.ts

```diff
@@ -79,6 +80,21 @@ export class OccCustomerTicketingAdapter implements CustomerTicketingAdapter {
       );
   }
 
+  createTicket(
```

Comment:
please add test for 200 and 401/500 types of responses

Reply:
I will include this in my next PR for tests